src/payment/strategies/paypal-commerce/paypal-commerce-sdk.ts

```diff
@@ -60,12 +61,162 @@ export interface ButtonsOptions {
     style?: PaypalButtonStyleOptions;
     fundingSource?: string;
     createOrder?(): Promise<string>;
-    onApprove?(data: ApproveDataOptions): void;
+    onApprove?(data: ApproveDataOptions, actions: ApproveActions): void;
+    onShippingChange?(data: ShippingChangeData, actions: ApproveActions): void;
     onClick?(data: ClickDataOptions, actions: ClickActions): void;
     onCancel?(): void;
     onError?(error: Error): void;
 }
 
+export interface ApproveActions {
+    order: {
+        capture(): Promise<PayerDetails>;
+        authorize(): Promise<PayerDetails>;
+        patch(data: PatchArgument[]): void;
+    };
+    resolve(): void;
+    reject(): void;
+}
+
+export interface CheckoutWithBillingAddress {
```

Comment:
Didn't find any usage of the interface. Can it be removed?

Reply:
removed